app/models/manageiq/providers/nuage/inventory/persister.rb

```diff
@@ -0,0 +1,42 @@
+class ManageIQ::Providers::Nuage::Inventory::Persister < ManagerRefresh::Inventory::Persister
+  require_nested :NetworkManager
+
+  # TODO(lsmola) figure out a way to pass collector info, probably via target, then remove the below
+  attr_reader :collector
```

Comment:
unless we need the collector inside of the initialize_inventory_collections method, we can drop this and the initialize method. (will make it easier for me do do the refactoring later :-))

Reply:
yeah, I don't see we need this. Actually I might have already done that TODO work, just need to delete the TODOs :-D